server/src/main/java/org/jetbrains/bsp/bazel/server/diagnostics/DiagnosticsService.kt

```diff
@@ -1,16 +0,0 @@
-package org.jetbrains.bsp.bazel.server.diagnostics
-
-import ch.epfl.scala.bsp4j.PublishDiagnosticsParams
-import org.jetbrains.bsp.bazel.bazelrunner.BazelInfo
-
-class DiagnosticsService(bazelInfo: BazelInfo) {
```

Comment:
I would keep this class. We have ProjectSyncService and ExecutionService that handle bsp structures (inputs bsp and outputs bsp). They are responsible for mapping inputs, calling some internal independent api and map outputs. I wanted to keep this convention with diagnostics service. It is a small class, but just keeps things clean and consistent.

Reply:
yeah that's cool, I can add this back.